katz/src/main/kotlin/katz/instances/Composed.kt

```diff
@@ -0,0 +1,54 @@
+package katz
+
+/**
+ * https://www.youtube.com/watch?v=wvSP5qYiz4Y
+ */
+interface ComposedType<out F, out G>
```

Comment:
could it be private or break implicit magic?
My reason it's not expose it when you import kats that would confuse people that use it.